examples/karpenter/main.tf

```
@@ -137,6 +137,12 @@ module "eks_blueprints" {
       source_cluster_security_group = true
     }
   }
+
+  # Add karpenter.sh/discovery tag so that we can use this as securityGroupSelector in karpenter provisioner
+  node_security_group_tags = {
+    "karpenter.sh/discovery" = local.cluster_name
```

It may seem redundant, but this allows for multiple clusters within the same account without having tag conflicts. More info can be found [here](https://github.com/aws/karpenter/pull/1668)

```suggestion
   "karpenter.sh/discovery/${local.cluster_name}" = local.cluster_name
```

---

Done